### PR TITLE
Added statement of ethics sponsorships and new sponsorship

### PR DESCRIPTION
**Pull request by crutchcorn**
Talking with [The Polyglot Developer](https://thepolyglotdeveloper.com), it seemed that we were hoping to sponsor each other's content including showcasing content from the other on a case-by-case basis and showcasing one another's logos and links on our homepage. They've been a massive supporter of our work for some time now and I think it's overall a good idea

Bringing this up with the team, we wanted to ensure our morals were inline in prep for ANY sponsors. This PR does the following to support this:
- Added statement of ethics which includes mention of our CoC in our README
- Added said statement to our about us page

We've also:
- Added the Polyglot Developer logo to our README
- Added the Polyglot Developer logo to our about us page
- Added mention of a related podcast episode from The Polyglot Developer to a post

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,27 +17,62 @@ Integration branch status: [![Integration Branch Build Status](https://travis-ci
...
 This repository acts as the source code location for [the Unicorn Utterances blog](https://unicorn-utterances.com)
 
-## Important Files
+## Sponsors
+
+[![The Polyglot Developer](./static/sponsors/the-polyglot-developer.svg)](https://www.thepolyglotdeveloper.com/)
+
+## Statement of Ethics
+
+We never want to end up in a place where our educational content or the community or
```
Comment (crutchcorn) [inline README.md]: One of the concerns raised by @evelynhathaway:
> I would rephrase the "never [negative thing]" to "always [positive thing saying outright why it's good and hinting at why it's bad]"

Comment (crutchcorn) [reply README.md]: I would argue that in this particular instance
I think with ethics hard lines do need to be drawn occationally
And I think laying it out as the first item showcases that

Comment (crutchcorn) [inline README.md]: A note on tone from @adueppen:
> I feel like there are a few changes to the wording I would make besides that too but besides that I think it's good

File: README.md [hunk 2]
```diff
@@ -17,27 +17,62 @@ Integration branch status: [![Integration Branch Build Status](https://travis-ci
...
+experience around that content is compromised. Not by money and not by any potential
+of harmful members within said community. As a result,
+[we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
```
Comment (adueppen) [inline README.md]:
```suggestion
We never want to end up in a place where our educational content, the
experience around that content, or our community is compromised; neither by
financial sway or by any potentially harmful members of the community. As a
result, [we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
```

File: README.md [hunk 3]
```diff
@@ -17,27 +17,62 @@ Integration branch status: [![Integration Branch Build Status](https://travis-ci
...
+
+We also pledge to keep transparent communication in regards to finances
+that flow through the project. While not every sponsorship contains a
+financial contribution, if one does it will be disclosed both what those
+finances are going towards as well as what will be done in exchange for
+said contribution will be clearly and transparently laid out.
```
Comment (adueppen) [inline README.md]:
```suggestion
We also pledge to maintain transparency in regards to any finances that flow
through the project. Not every sponsorship contains a financial contribution,
but for ones that do, it will be clearly disclosed what those finances are
going towards as well as what will be done in exchange for that contribution.
```

File: README.md [hunk 4]
```diff
@@ -23,16 +23,15 @@ This repository acts as the source code location for [the Unicorn Utterances blo
...
-that flow through the project. While not every sponsorship contains a
-financial contribution, if one does it will be disclosed both what those
-finances are going towards as well as what will be done in exchange for
-said contribution will be clearly and transparently laid out.
+We never want to end up in a place where our educational content, the
+experience around that content, or our community is compromised; neither by
+financial sway or by any potentially harmful members of the community. As a
+result, [we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
+
+We also pledge to maintain transparency in regards to any finances that flow
```
Comment (crutchcorn) [inline README.md]:
```suggestion
We also pledge to maintain transparency in regards to any finances that flows
```

File: content/site/about-us.md [hunk 5]
```diff
@@ -15,4 +15,21 @@ We know this is a lofty goal, though, and we don't want to do it alone. If you'r
...
+## Sponsors
+
+<a href="https://www.thepolyglotdeveloper.com/" target="_blank" rel="noopener noreferrer sponsored"><img alt="The Polyglot Developer" src="/sponsors/the-polyglot-developer.svg" width="300"/></a>
+
+## Statement of Ethics {#ethics}
+
+We never want to end up in a place where our educational content or the community or
+experience around that content is compromised. Not by money and not by any potential
+of harmful members within said community. As a result,
+[we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
```
Comment (fennifith) [inline content/site/about-us.md]: Maybe update this to match the statement in the README? I'm not sure which is newer.

File: README.md [hunk 6]
```diff
@@ -17,27 +17,61 @@ Integration branch status: [![Integration Branch Build Status](https://travis-ci
...
+## Sponsors
+
+[![The Polyglot Developer](./static/sponsors/the-polyglot-developer.svg)](https://www.thepolyglotdeveloper.com/)
+
+## Statement of Ethics
+
+We never want to end up in a place where our educational content, the
+experience around that content, or our community is compromised; neither by
+financial sway or by any potentially harmful members of the community. As a
+result, [we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
```
Comment (fennifith) [inline README.md]: Suggested change:

```diff
- We never want to end up in a place where our educational content, the
- experience around that content, or our community is compromised; neither by
- financial sway or by any potentially harmful members of the community. As a
- result, [we've implemented the Contributor Covenant as our code of conduct](CODE_OF_CONDUCT.md).
+ We never want to end up in a place where our educational content, experience,
+ or community is compromised by either financial sway or potentially harmful
+ members of the community. As such, we've implemented the
+ [Contributor Covenant](https://www.contributor-covenant.org/)
+ as our [code of conduct](CODE_OF_CONDUCT.md) to uphold these values.
```

Says basically the same thing, just more concise and a bit more readable (IMO) without a semicolon in the middle.

File: content/site/about-us.md (hunk 5 above)
Comment (crutchcorn) [reply content/site/about-us.md]: Oh right. Thank you

File: content/site/about-us.md [hunk 7]
```diff
@@ -21,15 +21,16 @@ If you’d like to get in touch with us, [our Discord](https://discord.gg/FMcvc6
...
-finances are going towards as well as what will be done in exchange for
-said contribution will be clearly and transparently laid out.
+We never want to end up in a place where our educational content, experience,
+or community is compromised by either financial sway or potentially harmful
+members of the community. As such, we've implemented the
+[Contributor Covenant](https://www.contributor-covenant.org/)
+as our [code of conduct](CODE_OF_CONDUCT.md) to uphold these values.
+
+We also pledge to maintain transparency in regards to any finances that flow
+This conversation was marked as resolved by crutchcorn
```
Comment (fennifith) [inline content/site/about-us.md]: _This conversation was marked as resolved by crutchcorn_

Comment (crutchcorn) [reply content/site/about-us.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (evelynhathaway) [reply README.md]: @crutchcorn, the above is my only note after reviewing the entire PR. If no one else agreed, it's resolved in my book.

I'm sorry I couldn't make the review window. I am glad we're doing this. 😄
